wiring/inc/spark_wiring_wifi.h

```diff
@@ -267,13 +269,15 @@ class WiFiClass : public NetworkClass
 
     int getCredentials(WiFiAccessPoint* results, size_t result_count);
 
+#if !HAL_PLATFORM_NCP
     String hostname()
     {
         const size_t maxHostname = 64;
         char buf[maxHostname] = {0};
         network_get_hostname(*this, 0, buf, maxHostname, nullptr);
         return String(buf);
     }
+#endif
 
     int setHostname(const String& hostname)
```

Comment:
`setHostname()` is also not supported.

Reply:
👍 This returns a `SYSTEM_ERROR_NOT_SUPPORTED` though, once compiled.

https://github.com/particle-iot/device-os/blob/4105ae862d01a57df14a306c842136645b3fc2b2/hal/src/argon/wlan_hal.cpp#L389